prusa3d/Prusa-Link-Web
# Add typescript into the project

This PR:

- Adds typescript to the project
- Turns index.js into index.ts and add types necessary to run the module
- Adds possibility to import css into typescript modules
- Makes no functional changes

What to test:

1. Build project
2. Deploy target static files onto a printer
3. Make sure UI works
